app/services/setting_registry.rb

```diff
@@ -182,7 +182,7 @@ def _add(name, category:, type:, default:, description:, full_name:, context:, v
 
   def _find_or_new_db_record(name)
     definition = find(name)
-    Setting.find_by(name: name) || _new_db_record(definition)
+    Setting.find_by(name: name.to_s) || _new_db_record(definition)
```

Comment:
Could you help my understanding? Why is this needed?

Reply:
It is not, while debugging I was trying everything I could think of and this is a left over (that didn't help ofc) :)
I can get rid of this, but thought it's safer to always have a string just as a safety precaution :shrug:

Reply:
Reverted as you suggested, it might confuse future readers why we did this together in this patch :)